jobs/ci-kubernetes-e2e-gce-large-manual-up.env

```diff
@@ -0,0 +1,23 @@
+### provider-env
+CLOUDSDK_CORE_PRINT_UNHANDLED_TRACEBACKS=1
+
+### cluster-env
+NUM_NODES=1000
+NODE_SIZE=n1-standard-1
+NODE_DISK_SIZE=50GB
+CLUSTER_IP_RANGE=10.160.0.0/14
+LOGROTATE_MAX_SIZE=5G
+
+### kubernetes-env
+KUBE_OS_DISTRIBUTION=gci
+KUBE_ENABLE_CLUSTER_MONITORING=none
+# Allow slack for 1% of the nodes to be unready while up'ing cluster.
+ALLOWED_NOTREADY_NODES=10
+# Increase throughput in master components (to accommodate cluster scale).
+SCHEDULER_TEST_ARGS=--kube-api-qps=100
```

Comment:
I think we should remove those settings and use defaults. They should be set only if we're experimenting with changes that we want to make default.